### PR TITLE
docs: network migrations

### PR DESCRIPTION
**Pull request by KaiserKarel**
## Proposed changes
Added a section for bridgechains, with the first sections dedicated to network migrations. As we do not want people to migrate to v2.0, but instead jump to v2.1, I added an archive section with older migration guides.

Migration sections will contain multiple older.version -> latest.version guides, and every time we migrate to a newer version, requiring some guides/documentation, we can move the old guides to the archive. This prevents cluttering and ensures we do not need to support 15 bridgechains, all on their own versions.

- [X] Docs (documentation only changes)

## Checklist
- [X] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [X] I have read the [WRITING DOCUMENTATION](https://docs.ark.io/guidebook/contribution-guidelines/writing-documentation.html) guidelines
- [X] I have added necessary documentation

## Further comments
I think I properly documented the typescript changes, but @faustbrian will have to look over them again for correctness.

### CODE REVIEW

**Comment by ghost** [comment]
@KaiserKarel Thanks for submitting this pull request, a maintainer will get back to you shortly!

**Comment by ghost** [comment]
@faustbrian @supaiku0 @air1one - please review this pull request in the next few days. Be sure to explicitly select labels so I know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by ghost** [comment]
@KaiserKarel Your pull request doesn't follow our [contribution guidelines](https://docs.ark.io/guidebook/contribution-guidelines/). Please review and correct it.

File: docs/bridgechains/migrations/migrating_2.0_2.1.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+::: warning
+The following upgrading instructions are cumulative. That is if you want to upgrade from version A to version C and there is version B between A and C, you need to follow the instructions for both A and B.
+:::
+
+## Upgrade from Core 2.0.\* to 2.1.0
+
+- Run `yarn run upgrade` from the root of the repository.
+
+- Remove `"@arkecosystem/core-config": {},` from the `~/.config/ark-core/<network>/plugins.js` file.
```
Comment (faustbrian) [inline docs/bridgechains/migrations/migrating_2.0_2.1.md]: See https://github.com/ArkEcosystem/core/pull/2040/files

File: docs/bridgechains/migrations/migrating_2.0_2.1.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+- Rename `@arkecosystem/core-transaction-pool-mem` to `@arkecosystem/core-transaction-pool` in the `~/.config/ark-core/<network>/plugins.js` file.
```
Comment (faustbrian) [inline docs/bridgechains/migrations/migrating_2.0_2.1.md]: See https://github.com/ArkEcosystem/core/pull/2040/files

File: docs/bridgechains/migrations/migrating_2.0_2.1.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        timelockTransfer: 500,
+        multiPayment: 500,
+        delegateResignation: 400000,
+    },
+},
+```
+
+## Conclusion
+
+Once all these changes have been made you will need to restart your relay and forger _(if you are a delegate)_ for these changes to take effect.
```
Comment (faustbrian) [inline docs/bridgechains/migrations/migrating_2.0_2.1.md]: See https://github.com/ArkEcosystem/core/pull/2040/files

File: docs/bridgechains/migrations/migrating_2.0_2.1.md (hunk 2 above)
Comment (KaiserKarel) [reply docs/bridgechains/migrations/migrating_2.0_2.1.md]: fixed in d237b79

File: docs/bridgechains/migrations/migrating_2.0_2.1.md (hunk 3 above)
Comment (KaiserKarel) [reply docs/bridgechains/migrations/migrating_2.0_2.1.md]: fixed in d237b79

**Review by faustbrian — APPROVED** [review]

**Comment by ghost** [comment]
@KaiserKarel A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
@KaiserKarel Your pull request has been merged but was not assigned a bounty tier. @faustbrian @supaiku0 @air1one - please assign a bounty tier to this pull request in the next few days.

**Pull request merged**